common/djangoapps/entitlements/api/urls.py

```diff
@@ -1,5 +1,6 @@
 from django.conf.urls import include, url
 
+app_name = 'entitlements'
 urlpatterns = [
     url(r'^v1/', include('entitlements.api.v1.urls', namespace='v1')),
```

Comment:
After reading the [release notes](https://docs.djangoproject.com/en/1.11/releases/1.9/#passing-a-3-tuple-or-an-app-name-to-include), I think these `namespace` parameters to `include()` can be removed when they match `app_name` in the included module or tuple.

Reply:
Thanks, @jmbowman . I'll try removing those as well.